week2/homework/src/index.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+let fs = require('fs');
+
+let args = process.argv.slice(2);
+
+let command = args[0];
+let todoItem = args[1];
+
```

Comment:
All the above variables are not re-assigned, so `const` should be used to declare them.